webapp/src/components/training/Trainer.vue

```diff
@@ -8,30 +8,70 @@
     </div>
     <!-- Train Button -->
     <div class="flex justify-center">
-      <IconCard title-placement="center">
+      <IconCard title-placement="center" :fill-space="true">
```

Comment:
one of the nice thing that you have with boolean props is that you don't even need to set it to true, its mere presence is enough.

```suggestion
      <IconCard title-placement="center" fill-space>
```